app/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentDownloadController.java

```diff
@@ -66,16 +66,16 @@ public ExperimentDownloadController(ExperimentTrader experimentTrader,
                                                 proteomicsExperimentDownloadSupplier,
                                         ExperimentDownloadSupplier.RnaSeqBaseline
                                                 rnaSeqBaselineExperimentDownloadSupplier,
-                                        ExperimentDownloadSupplier.RnaSeqDifferential
-                                                rnaSeqDifferentialExperimentDownloadSupplier,
+                                        ExperimentDownloadSupplier.BulkDifferential
+                                                    bulkDifferential,
```

Comment:
This variable should be named `bulkDifferentialExperimentDownloadSupplier` to be consistent with the others.